### DISCUSSION TITLE
How to change AirSim standart drone mesh to my drone or standart car to my car

### DISCUSSION THREAD

**Discussion by Supaple-x** [microsoft/AirSim#3433] [Support Q&A] [answered]
### I have tried to find the actor of standart AirSimDrone and change it in ue. But I didn't find.
Do someone have any tutorial how to perform this customization?

Thanks

**Comment by ahmed-elsaharti** [comment, marked as answer]
Hi @Supaple-x , to change the drone/car mesh in AirSim you'd have to modify the AirSim uasset for the vehicle or create a similar one. To do so, have a look at #3190, [this guide](https://github.com/Microsoft/AirSim/blob/master/docs/settings.md#pawnpaths) and this video:
https://drive.google.com/file/d/1-iOfJ8zO3YA4Mzc0ZkKwTM3L4--4RFtt/view?usp=sharing

Note: changing the drone's pawn would only change the visual appearance of the vehicle. This does not change any physics aspects like type of motor or arm length (as far as the physics is concerned). To change those you would have to look into [MultiRotorParams](https://github.com/microsoft/AirSim/blob/master/AirLib/include/vehicles/multirotor/MultiRotorParams.hpp) and [RotorParams](https://github.com/microsoft/AirSim/blob/master/AirLib/include/vehicles/multirotor/RotorParams.hpp)

**Comment by Supaple-x** [comment]
Hi @ahmed-elsaharti ,
Thank you! This tutorial compleatly meets my targets.
Very detailed and accurate.
All the best!
